tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -471,6 +472,83 @@ export class WebGPUBackend extends KernelBackend {
     return vals;
   }
 
+  async downloadGPUBufferData(gpuData: GPUData):
```

Comment:
The function body is very similar with `getBufferData`. Can you write a common helper function to reuse the code?

Reply:
Done